install-boost.sh

```diff
@@ -10,8 +10,7 @@ BOOST_ROOT="${BOOST_ROOT=${RIME_ROOT}/thirdparty/src/boost_${boost_x_y_z}}"
 
 boost_tarball="boost_${boost_x_y_z}.tar.bz2"
 download_url="https://boostorg.jfrog.io/artifactory/main/release/${boost_version}/source/${boost_tarball}"
-boost_tarball_sha256sum_1_75_0='953db31e016db7bb207f11432bef7df100516eeb746843fa0486a222e3fd49cb  boost_1_75_0.tar.bz2'
-boost_tarball_sha256sum="${boost_tarball_sha256sum=${boost_tarball_sha256sum_1_75_0}}"
+boost_tarball_sha256sum="953db31e016db7bb207f11432bef7df100516eeb746843fa0486a222e3fd49cb  boost_${boost_x_y_z}.tar.bz2"
```

Comment:
this change is incorrect - the script would check the fixed checksum which is for boost 1.75, and user wouldn't be able to use a different version.

please revert this commit.